source/sass/views/youtube-regrets-reporter.scss

```diff
@@ -0,0 +1,314 @@
+#view-youtube-regrets-reporter {
```

Comment:
there's just a ton of overrides to deal with so the code here is not the cleanest 🙈